cleverhans/utils_tf.py

```diff
@@ -49,13 +50,18 @@ def model_loss(y, model, mean=True):
              sample loss
     """
 
-    op = model.op
```

Comment:
Depending on what we decide to do with `Model.get_loss`, this edit should be removed or the docstring improved if we decide to keep `get_loss`